Model/Sales/MagentoCollection.php

```diff
@@ -15,6 +15,7 @@
 use Magento\Framework\ObjectManagerInterface;
 use Magento\Sales\Model\Order;
 use MyParcelNL\magento\Model\Order\Email\Sender\TrackSender;
```

Comment:
Deze mag met een hoofdletter
```suggestion
use MyParcelNL\Magento\Model\Order\Email\Sender\TrackSender;
```